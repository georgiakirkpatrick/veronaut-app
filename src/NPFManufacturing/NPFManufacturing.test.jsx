import React from 'react'
import { render, screen } from '@testing-library/react'
import { describe, it, expect } from 'vitest'
import NPFManufacturing from './NPFManufacturing'

describe('NPFManufacturing', () => {
    it ('renders without crashing', () => {
        render(<NPFManufacturing />)
        screen.debug()
    })

    it ('render the UI as expected', () => {
        const tree = render(<NPFManufacturing />)
        expect(tree).toMatchSnapshot()
    })
})