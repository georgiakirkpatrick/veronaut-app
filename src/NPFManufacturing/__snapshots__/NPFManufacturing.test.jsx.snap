// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NPFManufacturing > render the UI as expected 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="manufacturing"
      >
        <section
          class="FormPage"
        >
          <h1>
            Manufacturing
          </h1>
          <form
            autocomplete="off"
            id="form-page"
          >
            <div
              class="FormPromptWithSub"
            >
              <p
                class="FormPromptWithSub__prompt"
              >
                Enter the manufacturing information provided on the product page.
              </p>
              <p
                class="FormPromptWithSub__subtitle"
              >
                If the product page does not provide factory names or locations, select "not disclosed" from the dropdown.
              </p>
            </div>
            <fieldset
              class="FormFieldset"
            >
              <h2>
                Sewing factory
              </h2>
              <div
                class="FormDropdown"
              >
                <label
                  for="sewing-location"
                >
                  Location
                </label>
                <select
                  name="countryId"
                  required=""
                />
              </div>
              <div
                class="FormDropdown"
              >
                <label
                  for="sewing-factory"
                >
                  Factory
                </label>
                <select
                  name="factoryId"
                  required=""
                >
                  <option
                    id="0"
                    value="0"
                  >
                    Select a factory
                  </option>
                  <option
                    id="1"
                    value="1"
                  >
                    Not disclosed
                  </option>
                </select>
              </div>
              <div
                class="FormPromptWithSub"
              >
                <p
                  class="FormPromptWithSub__prompt"
                >
                  Factory not listed above?
                </p>
                <p
                  class="FormPromptWithSub__subtitle"
                />
              </div>
              <button
                class="FormButton"
                type="button"
              >
                Add a factory
              </button>
            </fieldset>
            <fieldset
              class="FormFieldset"
            >
              <h2>
                Cutting factory
              </h2>
              <div
                class="FormDropdown"
              >
                <label
                  for="cutting-location"
                >
                  Location
                </label>
                <select
                  name="countryId"
                  required=""
                />
              </div>
              <div
                class="FormDropdown"
              >
                <label
                  for="cutting-factory"
                >
                  Factory
                </label>
                <select
                  name="factoryId"
                  required=""
                >
                  <option
                    id="0"
                    value="0"
                  >
                    Select a factory
                  </option>
                  <option
                    id="1"
                    value="1"
                  >
                    Not disclosed
                  </option>
                </select>
              </div>
              <div
                class="FormPromptWithSub"
              >
                <p
                  class="FormPromptWithSub__prompt"
                >
                  Factory not listed above?
                </p>
                <p
                  class="FormPromptWithSub__subtitle"
                />
              </div>
              <button
                class="FormButton"
                type="button"
              >
                Add a factory
              </button>
            </fieldset>
            <fieldset
              class="FormFieldset"
            >
              <h2>
                Does the product have any of the following certifications?
              </h2>
              <div
                class="FormCheckboxSection"
              >
                <div
                  class="empty"
                />
                <div
                  class="FormCheckboxSection__options-container"
                />
              </div>
              <div
                class="FormPromptWithSub"
              >
                <p
                  class="FormPromptWithSub__prompt"
                >
                  Certification not listed above?
                </p>
                <p
                  class="FormPromptWithSub__subtitle"
                />
              </div>
              <button
                class="FormButton"
                type="button"
              >
                Add a certification
              </button>
            </fieldset>
            <fieldset
              class="FormFieldset"
            >
              <div />
              <div
                class="FormTextarea"
              >
                <label
                  for="additional-notes"
                >
                  Are there any additional notes about how the product was cut, sewn, or finished?
                </label>
                <textarea
                  id="additional-notes"
                  name="cmtNotes"
                  required=""
                  rows="1"
                />
              </div>
            </fieldset>
          </form>
          <div />
        </section>
        <footer
          class="NPFFooter"
        >
          <button
            class="NPFFooter__previous"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-angle-left NPFFooter__left-arrow"
              data-icon="angle-left"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 320 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M41.4 233.4c-12.5 12.5-12.5 32.8 0 45.3l160 160c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L109.3 256 246.6 118.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0l-160 160z"
                fill="currentColor"
              />
            </svg>
            <span>
              PREVIOUS
            </span>
          </button>
          <button
            class="NPFFooter__next"
            type="button"
          >
            <span>
              NEXT
            </span>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-angle-right NPFFooter__right-arrow"
              data-icon="angle-right"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 320 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M278.6 233.4c12.5 12.5 12.5 32.8 0 45.3l-160 160c-12.5 12.5-32.8 12.5-45.3 0s-12.5-32.8 0-45.3L210.7 256 73.4 118.6c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0l160 160z"
                fill="currentColor"
              />
            </svg>
          </button>
        </footer>
        <section
          class="FormPopUp"
          id="addSewFact"
        >
          <button
            class="FormPopUp__close-button"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-times fa-lg "
              data-icon="times"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 384 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M342.6 150.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L192 210.7 86.6 105.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L146.7 256 41.4 361.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L192 301.3 297.4 406.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L237.3 256 342.6 150.6z"
                fill="currentColor"
              />
            </svg>
          </button>
          <header>
            <h1>
              New Factory
            </h1>
          </header>
          <div
            class="FormPromptWithSub"
          >
            <p
              class="FormPromptWithSub__prompt"
            >
              Add a factory
            </p>
            <p
              class="FormPromptWithSub__subtitle"
            >
              Include the factory website if it is provided on the product page.  Copy and paste all available information about the factory in the 'notes' field
            </p>
          </div>
          <div
            class="FormTextInput"
          >
            <label
              for="sew-factory-name"
            >
              Factory name
            </label>
            <input
              id="sew-factory-name"
              name="name"
              required=""
              type="text"
              value=""
            />
          </div>
          <div
            class="FormDropdown"
          >
            <label
              for="sew-factory-location"
            >
              Factory location.  Select 'Not disclosed' if not included on the product page.
            </label>
            <select
              name="countryId"
              required=""
            />
          </div>
          <div
            class="FormUrlInput"
          >
            <label
              for="sew-factory-website"
            >
              Factory website.  Leave blank if not included on the product page.
            </label>
            <input
              id="sew-factory-website"
              name="website"
              required=""
              type="url"
              value=""
            />
          </div>
          <div
            class="FormTextarea"
          >
            <label
              for="sew-factory-notes"
            >
              Are there any additional notes about the sewing factory provided on the product page?  If so, copy them and paste them here.
            </label>
            <textarea
              id="sew-factory-notes"
              name="sewNotes"
              required=""
              rows="1"
            />
          </div>
          <button
            class="FormButton"
            type="submit"
          >
            SUBMIT FACTORY
          </button>
        </section>
        <section
          class="FormPopUp"
          id="addCutFact"
        >
          <button
            class="FormPopUp__close-button"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-times fa-lg "
              data-icon="times"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 384 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M342.6 150.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L192 210.7 86.6 105.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L146.7 256 41.4 361.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L192 301.3 297.4 406.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L237.3 256 342.6 150.6z"
                fill="currentColor"
              />
            </svg>
          </button>
          <header>
            <h1>
              New Factory
            </h1>
          </header>
          <div
            class="FormPromptWithSub"
          >
            <p
              class="FormPromptWithSub__prompt"
            >
              Add a factory
            </p>
            <p
              class="FormPromptWithSub__subtitle"
            >
              Include the factory website if it is provided on the product page.
            </p>
          </div>
          <div
            class="FormTextInput"
          >
            <label
              for="cut-factory-name"
            >
              Factory name
            </label>
            <input
              id="cut-factory-name"
              name="name"
              required=""
              type="text"
              value=""
            />
          </div>
          <div
            class="FormDropdown"
          >
            <label
              for="cut-factory-location"
            >
              Factory location.  Select 'Not disclosed' if not included on the product page.
            </label>
            <select
              name="countryId"
              required=""
            />
          </div>
          <div
            class="FormUrlInput"
          >
            <label
              for="cut-factory-website"
            >
              Factory website.  Leave blank if not included on the product page.
            </label>
            <input
              id="cut-factory-website"
              name="website"
              required=""
              type="url"
              value=""
            />
          </div>
          <div
            class="FormTextarea"
          >
            <label
              for="cut-factory-notes"
            >
              Are there any additional notes about the cutting factory provided on the product page?
            </label>
            <textarea
              id="cut-factory-notes"
              name="cutNotes"
              required=""
              rows="1"
            />
          </div>
          <button
            class="FormButton"
            type="submit"
          >
            SUBMIT FACTORY
          </button>
        </section>
        <section
          class="FormPopUp"
          id="cert-pop-up"
        >
          <button
            class="FormPopUp__close-button"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-times fa-lg "
              data-icon="times"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 384 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M342.6 150.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L192 210.7 86.6 105.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L146.7 256 41.4 361.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L192 301.3 297.4 406.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L237.3 256 342.6 150.6z"
                fill="currentColor"
              />
            </svg>
          </button>
          <header>
            <h1>
              New Certification
            </h1>
          </header>
          <div
            class="FormPromptWithSub"
          >
            <p
              class="FormPromptWithSub__prompt"
            >
              Add a certification
            </p>
            <p
              class="FormPromptWithSub__subtitle"
            />
          </div>
          <div
            class="FormTextInput"
          >
            <label
              for="man-cert-name"
            >
              Certification name
            </label>
            <input
              id="man-cert-name"
              name="name"
              required=""
              type="text"
              value="1"
            />
          </div>
          <div
            class="FormUrlInput"
          >
            <label
              for="man-cert-website"
            >
              Website.  Leave blank if not included on the product page.
            </label>
            <input
              id="man-cert-website"
              name="website"
              required=""
              type="url"
              value=""
            />
          </div>
          <button
            class="FormButton"
            type="submit"
          >
            Create certification
          </button>
        </section>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="manufacturing"
    >
      <section
        class="FormPage"
      >
        <h1>
          Manufacturing
        </h1>
        <form
          autocomplete="off"
          id="form-page"
        >
          <div
            class="FormPromptWithSub"
          >
            <p
              class="FormPromptWithSub__prompt"
            >
              Enter the manufacturing information provided on the product page.
            </p>
            <p
              class="FormPromptWithSub__subtitle"
            >
              If the product page does not provide factory names or locations, select "not disclosed" from the dropdown.
            </p>
          </div>
          <fieldset
            class="FormFieldset"
          >
            <h2>
              Sewing factory
            </h2>
            <div
              class="FormDropdown"
            >
              <label
                for="sewing-location"
              >
                Location
              </label>
              <select
                name="countryId"
                required=""
              />
            </div>
            <div
              class="FormDropdown"
            >
              <label
                for="sewing-factory"
              >
                Factory
              </label>
              <select
                name="factoryId"
                required=""
              >
                <option
                  id="0"
                  value="0"
                >
                  Select a factory
                </option>
                <option
                  id="1"
                  value="1"
                >
                  Not disclosed
                </option>
              </select>
            </div>
            <div
              class="FormPromptWithSub"
            >
              <p
                class="FormPromptWithSub__prompt"
              >
                Factory not listed above?
              </p>
              <p
                class="FormPromptWithSub__subtitle"
              />
            </div>
            <button
              class="FormButton"
              type="button"
            >
              Add a factory
            </button>
          </fieldset>
          <fieldset
            class="FormFieldset"
          >
            <h2>
              Cutting factory
            </h2>
            <div
              class="FormDropdown"
            >
              <label
                for="cutting-location"
              >
                Location
              </label>
              <select
                name="countryId"
                required=""
              />
            </div>
            <div
              class="FormDropdown"
            >
              <label
                for="cutting-factory"
              >
                Factory
              </label>
              <select
                name="factoryId"
                required=""
              >
                <option
                  id="0"
                  value="0"
                >
                  Select a factory
                </option>
                <option
                  id="1"
                  value="1"
                >
                  Not disclosed
                </option>
              </select>
            </div>
            <div
              class="FormPromptWithSub"
            >
              <p
                class="FormPromptWithSub__prompt"
              >
                Factory not listed above?
              </p>
              <p
                class="FormPromptWithSub__subtitle"
              />
            </div>
            <button
              class="FormButton"
              type="button"
            >
              Add a factory
            </button>
          </fieldset>
          <fieldset
            class="FormFieldset"
          >
            <h2>
              Does the product have any of the following certifications?
            </h2>
            <div
              class="FormCheckboxSection"
            >
              <div
                class="empty"
              />
              <div
                class="FormCheckboxSection__options-container"
              />
            </div>
            <div
              class="FormPromptWithSub"
            >
              <p
                class="FormPromptWithSub__prompt"
              >
                Certification not listed above?
              </p>
              <p
                class="FormPromptWithSub__subtitle"
              />
            </div>
            <button
              class="FormButton"
              type="button"
            >
              Add a certification
            </button>
          </fieldset>
          <fieldset
            class="FormFieldset"
          >
            <div />
            <div
              class="FormTextarea"
            >
              <label
                for="additional-notes"
              >
                Are there any additional notes about how the product was cut, sewn, or finished?
              </label>
              <textarea
                id="additional-notes"
                name="cmtNotes"
                required=""
                rows="1"
              />
            </div>
          </fieldset>
        </form>
        <div />
      </section>
      <footer
        class="NPFFooter"
      >
        <button
          class="NPFFooter__previous"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-angle-left NPFFooter__left-arrow"
            data-icon="angle-left"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 320 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M41.4 233.4c-12.5 12.5-12.5 32.8 0 45.3l160 160c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L109.3 256 246.6 118.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0l-160 160z"
              fill="currentColor"
            />
          </svg>
          <span>
            PREVIOUS
          </span>
        </button>
        <button
          class="NPFFooter__next"
          type="button"
        >
          <span>
            NEXT
          </span>
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-angle-right NPFFooter__right-arrow"
            data-icon="angle-right"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 320 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M278.6 233.4c12.5 12.5 12.5 32.8 0 45.3l-160 160c-12.5 12.5-32.8 12.5-45.3 0s-12.5-32.8 0-45.3L210.7 256 73.4 118.6c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0l160 160z"
              fill="currentColor"
            />
          </svg>
        </button>
      </footer>
      <section
        class="FormPopUp"
        id="addSewFact"
      >
        <button
          class="FormPopUp__close-button"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-times fa-lg "
            data-icon="times"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 384 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M342.6 150.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L192 210.7 86.6 105.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L146.7 256 41.4 361.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L192 301.3 297.4 406.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L237.3 256 342.6 150.6z"
              fill="currentColor"
            />
          </svg>
        </button>
        <header>
          <h1>
            New Factory
          </h1>
        </header>
        <div
          class="FormPromptWithSub"
        >
          <p
            class="FormPromptWithSub__prompt"
          >
            Add a factory
          </p>
          <p
            class="FormPromptWithSub__subtitle"
          >
            Include the factory website if it is provided on the product page.  Copy and paste all available information about the factory in the 'notes' field
          </p>
        </div>
        <div
          class="FormTextInput"
        >
          <label
            for="sew-factory-name"
          >
            Factory name
          </label>
          <input
            id="sew-factory-name"
            name="name"
            required=""
            type="text"
            value=""
          />
        </div>
        <div
          class="FormDropdown"
        >
          <label
            for="sew-factory-location"
          >
            Factory location.  Select 'Not disclosed' if not included on the product page.
          </label>
          <select
            name="countryId"
            required=""
          />
        </div>
        <div
          class="FormUrlInput"
        >
          <label
            for="sew-factory-website"
          >
            Factory website.  Leave blank if not included on the product page.
          </label>
          <input
            id="sew-factory-website"
            name="website"
            required=""
            type="url"
            value=""
          />
        </div>
        <div
          class="FormTextarea"
        >
          <label
            for="sew-factory-notes"
          >
            Are there any additional notes about the sewing factory provided on the product page?  If so, copy them and paste them here.
          </label>
          <textarea
            id="sew-factory-notes"
            name="sewNotes"
            required=""
            rows="1"
          />
        </div>
        <button
          class="FormButton"
          type="submit"
        >
          SUBMIT FACTORY
        </button>
      </section>
      <section
        class="FormPopUp"
        id="addCutFact"
      >
        <button
          class="FormPopUp__close-button"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-times fa-lg "
            data-icon="times"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 384 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M342.6 150.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L192 210.7 86.6 105.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L146.7 256 41.4 361.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L192 301.3 297.4 406.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L237.3 256 342.6 150.6z"
              fill="currentColor"
            />
          </svg>
        </button>
        <header>
          <h1>
            New Factory
          </h1>
        </header>
        <div
          class="FormPromptWithSub"
        >
          <p
            class="FormPromptWithSub__prompt"
          >
            Add a factory
          </p>
          <p
            class="FormPromptWithSub__subtitle"
          >
            Include the factory website if it is provided on the product page.
          </p>
        </div>
        <div
          class="FormTextInput"
        >
          <label
            for="cut-factory-name"
          >
            Factory name
          </label>
          <input
            id="cut-factory-name"
            name="name"
            required=""
            type="text"
            value=""
          />
        </div>
        <div
          class="FormDropdown"
        >
          <label
            for="cut-factory-location"
          >
            Factory location.  Select 'Not disclosed' if not included on the product page.
          </label>
          <select
            name="countryId"
            required=""
          />
        </div>
        <div
          class="FormUrlInput"
        >
          <label
            for="cut-factory-website"
          >
            Factory website.  Leave blank if not included on the product page.
          </label>
          <input
            id="cut-factory-website"
            name="website"
            required=""
            type="url"
            value=""
          />
        </div>
        <div
          class="FormTextarea"
        >
          <label
            for="cut-factory-notes"
          >
            Are there any additional notes about the cutting factory provided on the product page?
          </label>
          <textarea
            id="cut-factory-notes"
            name="cutNotes"
            required=""
            rows="1"
          />
        </div>
        <button
          class="FormButton"
          type="submit"
        >
          SUBMIT FACTORY
        </button>
      </section>
      <section
        class="FormPopUp"
        id="cert-pop-up"
      >
        <button
          class="FormPopUp__close-button"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-times fa-lg "
            data-icon="times"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 384 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M342.6 150.6c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L192 210.7 86.6 105.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L146.7 256 41.4 361.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0L192 301.3 297.4 406.6c12.5 12.5 32.8 12.5 45.3 0s12.5-32.8 0-45.3L237.3 256 342.6 150.6z"
              fill="currentColor"
            />
          </svg>
        </button>
        <header>
          <h1>
            New Certification
          </h1>
        </header>
        <div
          class="FormPromptWithSub"
        >
          <p
            class="FormPromptWithSub__prompt"
          >
            Add a certification
          </p>
          <p
            class="FormPromptWithSub__subtitle"
          />
        </div>
        <div
          class="FormTextInput"
        >
          <label
            for="man-cert-name"
          >
            Certification name
          </label>
          <input
            id="man-cert-name"
            name="name"
            required=""
            type="text"
            value="1"
          />
        </div>
        <div
          class="FormUrlInput"
        >
          <label
            for="man-cert-website"
          >
            Website.  Leave blank if not included on the product page.
          </label>
          <input
            id="man-cert-website"
            name="website"
            required=""
            type="url"
            value=""
          />
        </div>
        <button
          class="FormButton"
          type="submit"
        >
          Create certification
        </button>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
